### ISSUE TITLE
Web interface progress monitoring for DataSources and DataSinks with splits

### ISSUE THREAD

**Issue by rmetzger** [stratosphere/stratosphere#766]
The progress monitoring for DataSources and DataSinks can be improved by including the number of processed vs total splits into the progress.

**Comment by uce** [comment]
+1

**Comment by StephanEwen** [comment]
+1

**Comment by fhueske** [comment]
:+1:

**Comment by rmetzger** [comment]
I think @tobwiens is working on this. Are there any updates?

**Comment by tobwiens** [comment]
Yes, since I am working on the interface I will fix both issues. 

Current progress: The input splits of the data source are correctly displayed. After the page is displayed for the first time, an update routine is shifting the progress bars. The update routine works completely different  and makes it a struggle.
